public/components/trace_analytics/components/traces/traces_content.tsx

```diff
@@ -95,43 +101,40 @@ export function TracesContent(props: TracesProps) {
 
     setSortingColumns(sortColumns);
 
+    const localOnlyFields = ['trace_group', 'percentile_in_trace_group', 'trace_id'];
```

Comment:
maybe add some comments to explain what local only means, for future reference

Reply:
Added comment
// The columns that can not be used in a query, only rendered on page